recipe/meta.yaml

```
@@ -1,23 +1,24 @@
 {% set name = "miniopy-async" %}
-{% set version = "1.21.1" %}
+{% set version = "1.21.2" %}
```

```suggestion
{% set version = "1.21.2" %}
{% set python_min = "3.9" %}
```

Probably also worth explicitly setting your `python_min` to match that of your  lower bound in `pyproject.toml` for the reasons outlined in the "Hints" in https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python.

---

Ah, actually you're using Python 3.11 features 

```
import: 'miniopy_async'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/miniopy-async_1743142956513/test_tmp/run_test.py", line 2, in <module>
    import miniopy_async
  File "/home/conda/feedstock_root/build_artifacts/miniopy-async_1743142956513/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/miniopy_async/__init__.py", line 57, in <module>
    from .api import Minio
  File "/home/conda/feedstock_root/build_artifacts/miniopy-async_1743142956513/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/miniopy_async/api.py", line 51, in <module>
    from . import __title__, __version__, time
  File "/home/conda/feedstock_root/build_artifacts/miniopy-async_1743142956513/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.9/site-packages/miniopy_async/time.py", line 26, in <module>
    from datetime import UTC, datetime, timezone
ImportError: cannot import name 'UTC' from 'datetime'
```

so your upstream metadata in https://github.com/hlf20010508/miniopy-async is wrong. `UTC` was added in Python `3.11`, so you should fix that here.

---

Really appreciated.
3.10 is fine, I did version judgement.

---

No, c.f. https://docs.python.org/3/library/datetime.html#datetime.UTC